src/relations.d.ts

```diff
@@ -14,3 +14,8 @@ export type RelationTypeFunction = (attribute: string) => string;
  * The relationTypes option can be a function or an object
  */
 export type RelationTypeOpt = RelationTypeMap | RelationTypeFunction;
+
+export interface RelationOpts {
```

Comment:
I just changed this and it broke a bunch of things. That said, in the interest of time, I'm just going to leave this as-is. Seems more of a cosmetic change than anything, no?